openapi/platform/platform.yaml

```diff
@@ -2933,12 +2933,24 @@ paths:
                 comments:
                   type: string
                   description: more description for cancellation of products
+                deactivationType:
+                  type: string
+                  description: |-
+                    `deactivationTypeCancel` Cancel indicates we should turn the item off at the items anniversary date, or commitment date, whichever is later
```

Comment:
```suggestion
                    `cancel` Cancel indicates we should turn the product or service off at the anniversary date, or commitment date, whichever is later and till then it will be available for use.
```
I feel like we don't have to append `deactivationType` since we are already in the deactivationType block 💭 

@richard-rance Any thoughts on this?

Reply:
I would agree. 
In protos we add the enum type because we generate global constants in the SDKs for each option. The value of those constants are just ints. Here we are just defining the value to be a string.